### ISSUE TITLE
Swipe to right and to left is not working with Dragable View?

### ISSUE THREAD

**Issue by DawinderGill** [pedrovgs/DraggablePanel#98]
I am using dragable view.After swiping view at bottom i am not able to close view to swipe in left or right.Even c'nt able to open top view maximize with swiping from bottom. Can you please tell what to do to resolve this issue??

Thanks...

**Comment by pedrovgs** [comment]
Hi @DawinderGill to be able to help you I need more information:
- Version of the library you are using.
- Layout configured using the library.
- Code initializing the library.
- Version of the Android SDK used to build the APK and the target sdk configured.

The easiest way for providing this information is to create a sample project reproducing the bug :)

**Comment by DawinderGill** [comment]
Hi
Thanks for reply.
That issue is resolved but these are some new issues.
1. I am using video view in top dragable view. Can you please tell me how
to make this video view fullscreen on button click because it creates
problems when we make ot full screen.
2. Is the any option in this library to disable dragable method because i
want to disable dragable functionality when user on landscape mode.

Please reply me as soon as possible.
Thanks

**Comment by Blackquan** [comment]
Hi,
I got a same issue, can you tell me how to fix

For more info:
- compileSdkVersion 23
- buildToolsVersion "23.0.2"
- minSdkVersion 16
- targetSdkVersion 23
- compile 'com.github.pedrovgs:draggablepanel:1.9'

**Comment by DawinderGill** [comment]
Right now issue is not resolved. Will tell you if i got any solution.

**Comment by DawinderGill** [comment]
Yes @Blackquan for right or left wipe i am using dragable view and i have implement all again with reference of this library sample. Then this problem have been solved. You can also try this. :)

**Comment by Blackquan** [comment]
this library sample?

Can you explain in detail? Sorry i am newbie

**Comment by DawinderGill** [comment]
No problem bro.
Yes this sample.

**Comment by Blackquan** [comment]
i just did it, still not work for me :(

**Comment by DawinderGill** [comment]
You are using dependecy or import library??

**Comment by Blackquan** [comment]
already in 2 way :(

**Comment by DawinderGill** [comment]
Ok friend
I am not sure about this but this library having this issue. First time i
have also face this problem. After that i have implement 4th type in sample
i.e where they play spiderman movie trailer. Try that one once hope this
will be solved.
Otherwise you have to talk with support team.

**Comment by GaNhep** [comment]
I finally fixed this "bug", but don't know why this happen.
follwing the examples, if you have the top layout (A) and the bottom layout (B) inside the draggble view layout, there are 2 way to arrange them.
1: draggbleView(A, B) -> couldn't work
2: draggbleView(B,A) -> it works like a champ
can anyone explan why?

**Comment by tainangtk9** [comment]
> I finally fixed this "bug", but don't know why this happen.
follwing the examples, if you have the top layout (A) and the bottom layout (B) inside the draggble view layout, there are 2 way to arrange them.
1: draggbleView(A, B) -> couldn't work
2: draggbleView(B,A) -> it works like a champ
can anyone explan why?

it's working 👍  many thank you GaNhep!! i dont know why
